api/src/controllers/export-data.js

```diff
@@ -92,25 +91,28 @@ module.exports = {
       })
       .then(() => auth.check(req, getExportPermission(req.params.type)))
       .then(() => {
-        writeExportHeaders(res, req.params.type, formats.csv);
+        return new Promise(resolve => {
```

Comment:
I don't think this promise does anything - if I remove it everything works as expected. Can it be removed?